helm/ory-oathkeeper/templates/deployment.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "ory-oathkeeper.fullname" . }}
+  labels:
+{{ include "ory-oathkeeper.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "ory-oathkeeper.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "ory-oathkeeper.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      volumes:
+        - name: {{ include "ory-oathkeeper.name" . }}-config-volume
+          configMap:
+            name: {{ include "ory-oathkeeper.fullname" . }}
+        - name: {{ include "ory-oathkeeper.name" . }}-secrets-volume
+          secret:
+            secretName: {{ include "ory-oathkeeper.fullname" . }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: [ "oathkeeper", "serve", "--config", "/etc/config/config.yaml" ]
+          env:
+            {{- if .Values.oathkeeper.mutatorIdTokenJWKs }}
+            - name: MUTATORS_ID_TOKEN_JWKS_URL
+              value: "file:///etc/secrets/mutator.id_token.jwks.json"
+            {{- end }}
+            {{- if .Values.oathkeeper.accessRules }}
+            - name: ACCESS_RULES_REPOSITORIES
+              value: "file:///etc/config/access-rules.json"
+            {{- end }}
+          volumeMounts:
```

Comment:
I think (?) that's how it works. If you set `Values.oathkeeper.config.access_rules.repositories=http://remote/json` it should work. However, if you provide it via `oathkeeper.accessRules` directly, the above value will not work as it will be overwritten. I tried to document that in the README